plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-customers-v2-controller.php

```diff
@@ -55,7 +55,7 @@ protected function get_formatted_item_data_core( $object ) {
 		// Format date values.
 		foreach ( $format_date as $key ) {
 			// Date created is stored UTC, date modified is stored WP local time.
-			$datetime              = 'date_created' === $key ? get_date_from_gmt( gmdate( 'Y-m-d H:i:s', $data[ $key ]->getTimestamp() ) ) : $data[ $key ];
+			$datetime              = 'date_created' === $key && $data[ $key ] ? get_date_from_gmt( gmdate( 'Y-m-d H:i:s', $data[ $key ]->getTimestamp() ) ) : $data[ $key ];
```

Comment:
Nice!  This fixes the error on my end.  Nitpicky, but we might be able to make this more robust by checking if this is a `WC_DateTime` first or better yet, if it's any subclass of `DateTime`:

```suggestion
			$datetime              = 'date_created' === $key && is_subclass_of( $data[ $key ], 'DateTime' ) ? get_date_from_gmt( gmdate( 'Y-m-d H:i:s', $data[ $key ]->getTimestamp() ) ) : $data[ $key ];
```

That way we don't accidentally call `getTimestamp()` on something that may not have that method.